quixstreams/app.py

```diff
@@ -108,15 +112,25 @@ def __init__(
         loglevel: Optional[LogLevel] = "INFO",
         auto_create_topics: bool = True,
         use_changelog_topics: bool = True,
+        quix_config_builder: Optional[QuixKafkaConfigsBuilder] = None,
         topic_manager: Optional[TopicManager] = None,
     ):
         """
-
         :param broker_address: Kafka broker host and port in format `<host>:<port>`.
             Passed as `bootstrap.servers` to `confluent_kafka.Consumer`.
+            Either this OR quix_sdk_token must be set to use Application (not both).
+            Linked Environment Variable: `Quix__Broker__Address`
+            Default: None
+        :param quix_sdk_token: If using the Quix Platform, the SDK token to connect with.
+            Either this OR broker_address must be set to use Application (not both).
+            Linked Environment Variable: `Quix__Sdk__Token`
+            Default: None (if not run on Quix Platform)
+              >***NOTE:*** the environment variable is set for you on the Quix Platform
         :param consumer_group: Kafka consumer group.
             Passed as `group.id` to `confluent_kafka.Consumer`
-            Default - "quixstreams-default".
+            Linked Environment Variable: `Quix__Consumer__Group`
+            Default - "quixstreams-default" (post-init).
```

Comment:
What does "post-init" mean here?

Reply:
Was referring to the fact that `None` is not the true default here...but maybe it's enough to just say it's the default?

Reply:
Let's leave just "Default - ..." or explain how this default is figured out here.